### ISSUE TITLE
WaveOS Windows Installer

### ISSUE THREAD

**Issue by unclehowell** [unclehowell/WaveOS#11]
This is a feature request. Instead of having to download the .img file/ unzip/ burn to the SD Card etc  - it was suggested that a WaveOS tool be built (for Windows) which would do all those steps with a Wizard. This way you would simply insert the SD Card and download and run the Wizard. MAKES SENSE!

**Comment by unclehowell** [comment]
So I looked into this. There's programs like Install Creator, which make executable files more fancy. But we need something that actually builds upon Win32Disk Imager. So I stumbled across this repo - https://github.com/bizplay/win32-disk-imager  - which reverse engineers the program which writes the img file to the SD card. Perhaps this can be re branded and instead of letting the user select the directory of the file, it just looks to somewhere like github for the latest img file.  This is possibly the way to make the process easy for everyone. - on it!
